tests/filtertransformers/test_mongo.py

```diff
@@ -247,14 +264,37 @@ def test_operators(self):
             "$and": [
                 {"elements": {"$in": ["Ag"]}},
                 {
-                    "$and": [
-                        {"elements": {"$not": {"$in": ["Ir"]}}},
-                        {"elements": {"$not": {"$in": ["Ac"]}}},
+                    "$or": [
+                        {
+                            "$and": [
+                                {"elements": {"$nin": ["Ir"]}},
+                                {"elements": {"$ne": None}},
```

Comment:
If `$not -> $in` already does what we want around missing fields, why do we need to switch to $nin?

Reply:
It is indeed not strictly necessary to convert '$not: $in' to '$nin'.
I, however, find queries with $nin cleaner than queries with `$not:{$in:` , but this is probably also a matter of taste.
It also cleans up the filter, if there are two $not 's in front of in. 

If you want, I could add code that removes two sequential '$not's in general, although I think such queries will be rare.
This would also simplify filters with two $not's in front of statements like 'CONTAINS' and 'STARTS WITH', which are currently not cleaned up.

Reply:
It is not strictly necessary to switch to $nin. 
It however makes the filter a bit simpler to read, and the code was already there for $eq/$ne. (Although it is also not strictly necessary for $eq and $ne.)
In case of double negation `{a:{'$not':{'$not':{'$in:[1,2,3]}}}}` it will also simplify the filter to `{a:{'$in:[1,2,3]}}`.
This simplification is not yet implemented for filters that are not in inverse_operator_map because they do not have an opposite. For example: STARTS WITH  or CONTAINS. If you want, I can try to implement this simplification for these filters as well.